### ISSUE TITLE
sbs.com.au 24x7 livestream returns 403

### ISSUE THREAD

**Issue by wilsonssss** [yt-dlp/yt-dlp#5544]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.11.11** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

AU

### Provide a description that is worded well enough to be understood

```
 yt-dlp --allow-unplayable-formats --no-check-certificate --user-agent "KAIOS/2.0" "https://www.sbs.com.au/ondemand/watch/1726824003663"
```
SBS AU shows 403 error
![image](https://user-images.githubusercontent.com/4098951/201835256-607fb13e-f5e4-41ac-95c3-06c9015cdb83.png)

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
yt-dlp -vU --allow-unplayable-formats --no-check-certificate --user-agent "KAIOS/2.0" "https://www.sbs.com.au/ondemand/watch/1726824003663"                    13:37:18
[debug] Command-line config: ['-vU', '--allow-unplayable-formats', '--no-check-certificate', '--user-agent', 'KAIOS/2.0', 'https://www.sbs.com.au/ondemand/watch/1726824003663']
WARNING: You have asked for UNPLAYABLE formats to be listed/downloaded. This is a developer option intended for debugging.
         If you experience any issues while using this option, DO NOT open a bug report
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.09.01 [5d7c7d6] (pip)
[debug] Python 3.10.6 (CPython 64bit) - macOS-12.4-x86_64-i386-64bit
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 5.1.1 (setts), ffprobe 5.1.1, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {'all': 'socks5://127.0.0.1:7890', 'http': 'http://127.0.0.1:7890', 'https': 'http://127.0.0.1:7890'}
[debug] Loaded 1670 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.11.11, Current version: 2022.09.01
[debug] Downloading _update_spec from https://github.com/yt-dlp/yt-dlp/releases/download/2022.11.11/_update_spec
ERROR: You installed yt-dlp with pip or using the wheel from PyPi; Use that to update
[debug] [SBS] Extracting URL: https://www.sbs.com.au/ondemand/watch/1726824003663
[SBS] 1726824003663: Downloading JSON metadata
[debug] [ThePlatform] Extracting URL: http://link.theplatform.com/s/Bgtm9B/LA8bCeXVGLVU?feed=Video%20-%20Single&mbr=true&manifest=m3u&ord=493790&policy=11929623&dfptag=sz%3D530x298%26iu%3D%2F4117%2Fvideo.livestream.sbs.com.au%2Flivestreamhtmlweb%26ciu_szs%26impl%3Ds%26gdfp_req%3D1%26env%3Dvp%26output%3Dxml_vast3%26unviewed_position_start%3D1%26url%3Dwww.sbs.com.au%26description_url%3DSBS%26cust_params%3Dtype%253Dpreroll%26ad_rule%3D0%26cmsid%3D531%26url%3Dhttp%253A%252F%252Fwww.sbs.com.au%252Fondemand%252Fvideo%252Fsingle%252F1726824003663%26description_url%3DSBS%26correlator%3D--ORD--%26vid%3D1726824003663&dfpmidtag=sz%3D530x298%26iu%3D%2F4117%2Fvideo.livestream.sbs.com.au%2Flivestreamhtmlweb%26ciu_szs%26impl%3Ds%26gdfp_req%3D1%26env%3Dvp%26output%3Dxml_vast3%26unviewed_position_start%3D1%26url%3Dwww.sbs.com.au%26description_url%3DSBS%26cust_params%3Dtype%253Dpreroll%26ad_rule%3D0%26cmsid%3D531%26url%3Dhttp%253A%252F%252Fwww.sbs.com.au%252Fondemand%252Fvideo%252Fsingle%252F1726824003663%26description_url%3DSBS%26correlator%3D--ORD--%26vid%3D1726824003663#__youtubedl_smuggle=%7B%22force_smil_url%22%3A+true%7D
[ThePlatform] LA8bCeXVGLVU: Downloading SMIL data
[ThePlatform] LA8bCeXVGLVU: Downloading m3u8 information
WARNING: [ThePlatform] Failed to download m3u8 information: HTTP Error 403: Forbidden
[ThePlatform] LA8bCeXVGLVU: Downloading JSON metadata
ERROR: [ThePlatform] LA8bCeXVGLVU: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/usr/local/Cellar/yt-dlp/2022.9.1/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1459, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/Cellar/yt-dlp/2022.9.1/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1556, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/usr/local/Cellar/yt-dlp/2022.9.1/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1667, in process_ie_result
    return self.process_ie_result(
  File "/usr/local/Cellar/yt-dlp/2022.9.1/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1614, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
  File "/usr/local/Cellar/yt-dlp/2022.9.1/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 2546, in process_video_result
    self.raise_no_formats(info_dict)
  File "/usr/local/Cellar/yt-dlp/2022.9.1/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1025, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [ThePlatform] LA8bCeXVGLVU: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
```

**Comment by section83** [comment]
Can confirm this error on **all** SBS live stream URLs using YT-DLP version: 2022.11.11.

As I'm Oz-resident, can test commits which address this error.

**Comment by pukkandan** [comment]
Can you give log of just `yt-dlp https://www.sbs.com.au/ondemand/watch/1726824003663` without the problematic arguments in OP?

**Comment by section83** [comment]
Hope this helps:

> yt-dlp -v "https://www.sbs.com.au/ondemand/watch/1726824003663"
[debug] Command-line config: ['-v', 'https://www.sbs.com.au/ondemand/watch/1726824003663']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.11.11 [8b64402] (darwin_exe)
[debug] Python 3.8.9 (CPython arm64 64bit) - macOS-13.0.1-arm64-arm-64bit (LibreSSL 2.8.3)
[debug] exe versions: ffmpeg 5.1.2-https (setts), ffprobe 5.1.2-https
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.09.24, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1723 extractors
[debug] [SBS] Extracting URL: https://www.sbs.com.au/ondemand/watch/1726824003663
[SBS] 1726824003663: Downloading JSON metadata
[debug] [ThePlatform] Extracting URL: http://link.theplatform.com/s/Bgtm9B/LA8bCeXVGLVU?feed=Video%20-%20Single&mbr=true&manifest=m3u&ord=1395911&policy=11929623&dfptag=sz%3D530x298%26iu%3D%2F4117%2Fvideo.livestream.sbs.com.au%2Flivestreamhtmlweb%26ciu_szs%26impl%3Ds%26gdfp_req%3D1%26env%3Dvp%26output%3Dxml_vast3%26unviewed_position_start%3D1%26url%3Dwww.sbs.com.au%26description_url%3DSBS%26cust_params%3Dtype%253Dpreroll%26ad_rule%3D0%26cmsid%3D531%26url%3Dhttp%253A%252F%252Fwww.sbs.com.au%252Fondemand%252Fvideo%252Fsingle%252F1726824003663%26description_url%3DSBS%26correlator%3D--ORD--%26vid%3D1726824003663&dfpmidtag=sz%3D530x298%26iu%3D%2F4117%2Fvideo.livestream.sbs.com.au%2Flivestreamhtmlweb%26ciu_szs%26impl%3Ds%26gdfp_req%3D1%26env%3Dvp%26output%3Dxml_vast3%26unviewed_position_start%3D1%26url%3Dwww.sbs.com.au%26description_url%3DSBS%26cust_params%3Dtype%253Dpreroll%26ad_rule%3D0%26cmsid%3D531%26url%3Dhttp%253A%252F%252Fwww.sbs.com.au%252Fondemand%252Fvideo%252Fsingle%252F1726824003663%26description_url%3DSBS%26correlator%3D--ORD--%26vid%3D1726824003663#__youtubedl_smuggle=%7B%22force_smil_url%22%3A+true%7D
[ThePlatform] LA8bCeXVGLVU: Downloading SMIL data
[ThePlatform] LA8bCeXVGLVU: Downloading m3u8 information
WARNING: [ThePlatform] Failed to download m3u8 information: HTTP Error 403: Forbidden
[ThePlatform] LA8bCeXVGLVU: Downloading JSON metadata
ERROR: [ThePlatform] LA8bCeXVGLVU: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "yt_dlp/YoutubeDL.py", line 1485, in wrapper
  File "yt_dlp/YoutubeDL.py", line 1582, in __extract_info
  File "yt_dlp/YoutubeDL.py", line 1694, in process_ie_result
  File "yt_dlp/YoutubeDL.py", line 1641, in process_ie_result
  File "yt_dlp/YoutubeDL.py", line 2573, in process_video_result
  File "yt_dlp/YoutubeDL.py", line 1035, in raise_no_formats
yt_dlp.utils.ExtractorError: [ThePlatform] LA8bCeXVGLVU: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

**Comment by bashonly** [comment]
@section83 does this url give you 403 too?
https://sbs-live.akamaized.net/hls/live/2002827/channel1/geotok/master.m3u8

**Comment by wilsonssss** [comment]
let me try 2022.11.11, updating.

**Comment by section83** [comment]
Looks like same problem:

> yt-dlp -v "https://sbs-live.akamaized.net/hls/live/2002827/channel1/geotok/master.m3u8"
[debug] Command-line config: ['-v', 'https://sbs-live.akamaized.net/hls/live/2002827/channel1/geotok/master.m3u8']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.11.11 [8b64402] (darwin_exe)
[debug] Python 3.8.9 (CPython arm64 64bit) - macOS-13.0.1-arm64-arm-64bit (LibreSSL 2.8.3)
[debug] exe versions: ffmpeg 5.1.2-https (setts), ffprobe 5.1.2-https
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.09.24, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1723 extractors
[debug] [generic] Extracting URL: https://sbs-live.akamaized.net/hls/live/2002827/channel1/geotok/master.m3u8
[generic] master: Downloading webpage
ERROR: [generic] Unable to download webpage: HTTP Error 403: Forbidden (caused by <HTTPError 403: 'Forbidden'>); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp/extractor/common.py", line 674, in extract
  File "yt_dlp/extractor/generic.py", line 2582, in _real_extract
  File "yt_dlp/extractor/common.py", line 825, in _request_webpage

>  File "yt_dlp/extractor/common.py", line 807, in _request_webpage
  File "yt_dlp/YoutubeDL.py", line 3692, in urlopen
  File "urllib/request.py", line 531, in open
  File "urllib/request.py", line 640, in http_response
  File "urllib/request.py", line 569, in error
  File "urllib/request.py", line 502, in _call_chain
  File "urllib/request.py", line 649, in http_error_default
urllib.error.HTTPError: HTTP Error 403: Forbidden

**Comment by wilsonssss** [comment]
` >> yt-dlp -vU https://www.sbs.com.au/ondemand/watch/1726824003663

[debug] Command-line config: ['-vU', 'https://www.sbs.com.au/ondemand/watch/1726824003663']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.11.11 [8b64402] (pip)
[debug] Python 3.10.8 (CPython arm64 64bit) - macOS-12.4-arm64-arm-64bit (OpenSSL 1.1.1s  1 Nov 2022)
[debug] exe versions: ffmpeg 5.0.1 (setts), ffprobe 5.0.1, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {'https': 'http://127.0.0.1:7890', 'http': 'http://127.0.0.1:7890', 'all': 'socks5://127.0.0.1:7890'}
[debug] Loaded 1723 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.11.11, Current version: 2022.11.11
yt-dlp is up to date (2022.11.11)
[debug] [SBS] Extracting URL: https://www.sbs.com.au/ondemand/watch/1726824003663
[SBS] 1726824003663: Downloading JSON metadata
[debug] [ThePlatform] Extracting URL: http://link.theplatform.com/s/Bgtm9B/LA8bCeXVGLVU?feed=Video%20-%20Single&mbr=true&manifest=m3u&ord=1302258&policy=11929623&dfptag=sz%3D530x298%26iu%3D%2F4117%2Fvideo.livestream.sbs.com.au%2Flivestreamhtmlweb%26ciu_szs%26impl%3Ds%26gdfp_req%3D1%26env%3Dvp%26output%3Dxml_vast3%26unviewed_position_start%3D1%26url%3Dwww.sbs.com.au%26description_url%3DSBS%26cust_params%3Dtype%253Dpreroll%26ad_rule%3D0%26cmsid%3D531%26url%3Dhttp%253A%252F%252Fwww.sbs.com.au%252Fondemand%252Fvideo%252Fsingle%252F1726824003663%26description_url%3DSBS%26correlator%3D--ORD--%26vid%3D1726824003663&dfpmidtag=sz%3D530x298%26iu%3D%2F4117%2Fvideo.livestream.sbs.com.au%2Flivestreamhtmlweb%26ciu_szs%26impl%3Ds%26gdfp_req%3D1%26env%3Dvp%26output%3Dxml_vast3%26unviewed_position_start%3D1%26url%3Dwww.sbs.com.au%26description_url%3DSBS%26cust_params%3Dtype%253Dpreroll%26ad_rule%3D0%26cmsid%3D531%26url%3Dhttp%253A%252F%252Fwww.sbs.com.au%252Fondemand%252Fvideo%252Fsingle%252F1726824003663%26description_url%3DSBS%26correlator%3D--ORD--%26vid%3D1726824003663#__youtubedl_smuggle=%7B%22force_smil_url%22%3A+true%7D
[ThePlatform] LA8bCeXVGLVU: Downloading SMIL data
[ThePlatform] LA8bCeXVGLVU: Downloading m3u8 information
WARNING: [ThePlatform] Failed to download m3u8 information: HTTP Error 403: Forbidden
[ThePlatform] LA8bCeXVGLVU: Downloading JSON metadata
ERROR: [ThePlatform] LA8bCeXVGLVU: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/yt-dlp/2022.11.11/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1485, in wrapper
    return func(self, *args, **kwargs)
  File "/opt/homebrew/Cellar/yt-dlp/2022.11.11/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1582, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/opt/homebrew/Cellar/yt-dlp/2022.11.11/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1694, in process_ie_result
    return self.process_ie_result(
  File "/opt/homebrew/Cellar/yt-dlp/2022.11.11/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1641, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
  File "/opt/homebrew/Cellar/yt-dlp/2022.11.11/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 2573, in process_video_result
    self.raise_no_formats(info_dict)
  File "/opt/homebrew/Cellar/yt-dlp/2022.11.11/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1035, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [ThePlatform] LA8bCeXVGLVU: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U


`

**Comment by wilsonssss** [comment]
@bashonly  pls help

**Comment by bashonly** [comment]
Resolved by 6a765f135ccb654861336ea27a2c1c24ea8e286f
